src/DevChatter.Bot.Core/Games/Hangman/HangmanGame.cs

```diff
@@ -164,10 +171,34 @@ public void AttemptToStartGame(IChatClient chatClient, ChatUser chatUser)
                 return;
             }
 
-            IsRunning = true;
+            ALL_LETTERS = _hangmanSettings.AllowedCharacters;
+            if (ALL_LETTERS != null)
```

Comment:
Maybe we disable the `HangmanCommand` if `ALL_LETTERS` is null, instead of doing this check here.